docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+
+networks:
+  helium:
+
+services:
+  gateway-rs:
+    image: helium/gateway-rs:v1.0.0-alpha.23
+    build:
+      context: https://github.com/helium/gateway-rs.git#v1.0.0-alpha.23
+    volumes:
+      - './gateway_key.bin:/etc/helium_gateway/gateway_key.bin:ro'
+    environment:
+      - GW_REGION=US915
+      - GW_LOG_LEVEL=info 
+      - GW_UPDATE_ENABLED=false
+    networks:
+      - helium
+    expose:
+      - 1680/udp
+      - 4467/tcp
+    ports:
+      - 1680:1680/udp
+      - 4467:4467/tcp
```

Comment:
I'm not sure what the right setup here is.. you _definitely_ do _not_ want 4467 (the local api) exposed to the open internet since it basically allows anyone to get your gateway to sign anything.

Reply:
I changed it to bind to localhost on the host machine.

Reply:
https://github.com/helium/gateway-rs/pull/227/commits/5760d9c21cc41a2d8fa6456f64689df6f73c96de